bin/clocal-gcp

```diff
@@ -3,18 +3,23 @@
 'use strict';
 
 const program = require('commander');
-
+const FunctionalArray = require('../src/serviceCounter/API')
 const commandsArray = require('../src/services/index').commands;
-
 program.version('1.0.0').description('Clocal GCP');
 
-const commandNameList = [];
-
-commandsArray.map(command => {
+let length = items.length;
+for(var i = 0; i < length;i++){
+  const ServicesCmdBash = require('../src/services/cli-commands/'+items[i]+'/cmd');
+  commandsArray = [ServicesCmdBash];
+  program.version(<Version>).description('Clocal GCP');
+  
+  const commandNameList = [FunctionalArray];
+  commandsArray.map(command => {
```

Comment:
You could use `forEach` instead of `map` because `map` is used to create new arrays out of existing ones
```suggestion
  commandsArray.forEach(command => {
```